packages/datagrid/src/components/datagrid.component.stories.js

```diff
@@ -210,3 +211,57 @@ storiesOf('Data/Datagrid/Datagrid', module)
 		</div>
 	))
 	.add('datagrid with immutable data', () => <ImmutableDataGrid />);
+storiesOf('Data/Datagrid/QualityBar', module)
+	.add('default', () => (
+		<QualityBar
+			style={{ position: 'inherit' }}
+			{...{
+				valid: {
+					total: 50,
+					percentage: 33,
+				},
+				empty: {
+					total: 50,
+					percentage: 33,
+				},
+				invalid: {
+					total: 50,
+					percentage: 34,
+				},
+			}}
+		/>
+	))
+	.add('default with button filter', () => (
+		<QualityBar
+			style={{ position: 'inherit' }}
+			{...{
+				valid: {
+					total: 50,
+					percentage: 33,
+					buttonFilter: {
+						onClick: () => console.log('filter valide value'),
```

Comment:
don't use console.log in stories, use "action" (look at other stories for examples)